include/dt-bindings/interrupt-controller/it8xxx2-wuc.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_INTERRUPT_CONTROLLER_IT8XXX2_WUC_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_INTERRUPT_CONTROLLER_IT8XXX2_WUC_H_
+
+/** WUC reserved register of reg property */
```

Comment:
missing: `dt-bindings/dt-util.h` (for `BIT`)

Reply:
Done